packages/types/src/spec.ts

```diff
@@ -15,3 +15,11 @@ export interface FoundSpec extends SpecFile {
   fileExtension: string
   specType: Cypress.CypressSpecType
 }
+
+export type SetSpecsFoundBySpecPattern = {
```

Comment:
Minor: I'd have this type live in `ProjectActions` since it's not something that will be reused